pkg/backends/calico/routes.go

```diff
@@ -47,18 +47,12 @@ type routeGenerator struct {
 	svcIndexer, epIndexer   cache.Indexer
 	svcRouteMap             map[string]map[string]bool
 	routeAdvertisementCount map[string]int
-	clusterCIDR             string
+	clusterCIDRs            []string
```

Comment:
This change to plural isn't strictly necessary, but aligns us with the externalIPs field and will be required when we want IPv6 support.

Reply:
Don't think I understand.  It is necessary now that the data model allows multiple cluster CIDRs, isn't it?